app/pages/order-task-list/contextCreator.js

```diff
@@ -1,8 +1,12 @@
 import neworderPageManifest from './neworder/manifest.json';
+import taskListManifest from './taskListManifest.json';
+import { generateTaskList } from './helpers/generateTaskList';
 import { baseUrl } from '../../config';
 
+
 export const getContext = ({ pageName }) => ({
   ...neworderPageManifest,
+  taskList: generateTaskList({ orderId: pageName, taskListManifest }),
```

Comment:
instead of pageName I just used orderId throughout and the id of a new order is 'neworder'

Reply:
👍